main.css

```diff
@@ -0,0 +1,112 @@
+body {
+  font-family: sans-serif;
+  margin: 0;
+}
+
+header {
+  background-color: #1c1a19;
+  color: white;
+  background-image: url("./Images/illustration-header-mobile.svg");
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+.nav-bar {
+  background-color: #3c3a39;
+  display: flex;
+  justify-content: space-between;
+  padding: 10px 25px;
+  align-items: center;
+}
+
+.header-title {
+  padding: 3rem 25px;
+  line-height: 1.2;
+}
+
+.header-title h1 {
+  color: #ff6b00;
+  font-size: 50px;
+  margin: 0;
+  padding: 0;
+}
```

Comment:
- [ ] Good job working on the styles but still the design is little different from the requirements. You are supposed to stick to the [figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1). Please place some padding between the toolbar (or header) and the hero section So your hero section covers almost 100% of the screen, Follow the design on the template as much as possible.

Please check the images below for reference
Your design

![InkedScreenshot (42)](https://user-images.githubusercontent.com/83803414/153264789-58cf5336-f732-4f23-a04e-11d6bc8b2a4b.jpg)

Figma design

![Screenshot (43)](https://user-images.githubusercontent.com/83803414/153265032-ca587dad-4dc4-4552-9690-ddd0a2bda6fb.png)